airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.31.2
```

Comment:
This can be done as part of [the CDK release process] to avoid clash with other branches. Same with the changes in airbyte-cdk/python/setup.py